src/stavenote.ts

```diff
@@ -925,46 +1008,47 @@ export class StaveNote extends StemmableNote {
    * Get the staff line and y value for the highest & lowest noteheads
    * @returns {noteHeadBounds}
    */
-  getNoteHeadBounds() {
+  getNoteHeadBounds(): StaveNoteHeadBounds {
+    if (!this.stave) throw new Vex.RERR('NoStave', 'No stave attached to this note.');
     // Top and bottom Y values for stem.
-    let yTop = null;
-    let yBottom = null;
-    let nonDisplacedX = null;
-    let displacedX = null;
+    let yTop: number = 100000;
+    let yBottom: number = -100000;
```

Comment:
Better to use `+Infinity` and `-Infinity` here.

Reply:
Ok